### PR TITLE
Update/update react native

### PR DESCRIPTION
**Pull request by xkevin190**
react native was updated and some libraries that gave security conflicts.

### CODE REVIEW

File: android/app/src/main/java/com/lochameshchat/MainApplication.java [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 
 import com.facebook.react.ReactApplication;
+import com.oblador.vectoricons.VectorIconsPackage;
+import com.oblador.vectoricons.VectorIconsPackage;
```
Comment (jeandudey) [inline android/app/src/main/java/com/lochameshchat/MainApplication.java:9]:
```suggestion
```

File: android/settings.gradle [hunk 2]
```diff
@@ -1,4 +1,8 @@
 rootProject.name = 'LochaMeshChat'
+include ':react-native-vector-icons'
+project(':react-native-vector-icons').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-vector-icons/android')
+include ':react-native-vector-icons'
+project(':react-native-vector-icons').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-vector-icons/android')
```
Comment (jeandudey) [inline android/settings.gradle:5]:
```suggestion
```

Duplicated

Comment (xkevin190) [reply android/settings.gradle:5]: this is necessary because if I remove this lib the icons can not show.

Comment (xkevin190) [reply android/settings.gradle:5]: I can see it xD. it was added two times.

**Review by jeandudey — APPROVED** [review]

File: __Mocks__/index.js [hunk 3]
```diff
@@ -77,18 +80,27 @@ jest.mock('react-native-sound', () => class Sound {
...
+
+jest.mock('@react-native-picker/picker', () => {
+  // code here
+});
 
 jest.mock('react-native-gesture-handler', () => {
   // code here
 });
 
+
```
Comment (luisan00) [inline __Mocks__/index.js:95]: extra line
```suggestion
```

File: __Mocks__/index.js [hunk 4]
```diff
@@ -77,18 +80,27 @@ jest.mock('react-native-sound', () => class Sound {
...
 jest.mock('react-navigation-stack', () => {
   // code here
 });
 
 
 jest.mock('react-native-permissions', () => mock);
 
+
```
Comment (luisan00) [inline __Mocks__/index.js:103]: same here
```suggestion
```

**Review by luisan00 — APPROVED** [review]

**Comment by luisan00** [comment]
The code is correct and functional, only some extra lines

**Pull request merged**